script.js

```diff
@@ -0,0 +1,111 @@
+    description: 'Discover achu and yellow soupe, explaine how it can cure some certain illness.',
+  },
+  {
+    img: './images/phone-view-icons/investors_03.png',
+    name: 'sohyeong Noh',
+    profession: 'Chef in Hotel-china-town',
+    description: 'coming directly from HongKong to give her advice and encourage of this new restaurant.',
+  },
+  {
+    img: './images/phone-view-icons/investors_04.png',
+    name: 'Julia Reda',
+    profession: 'Representing Chef in hotel-picasso in France',
+    description: 'One of the best chef crown by the president of France itself.',
+  },
+  {
+    img: './images/phone-view-icons/investors_05.png',
+    name: 'Lila Treticov',
+    profession: 'Secretary of chef in russia of the department of brevrage',
+    description: 'Contribution in the exporting and importing of goods and services.',
+  },
+  {
+    img: './images/phone-view-icons/investors_06.png',
+    name: 'Ryan Merkley',
+    profession: 'President in the chef sector in hotel-Torento',
+    description: 'Incharge of investigation of how food have to be cooked in a restaurant.',
+  },
+];
+
+const container = document.getElementById('container');
+
+function data() {
+  speakerData.forEach((speaker) => {
+    container.innerHTML += `<article class="investors-list">
+    <img src="${speaker.img}" alt="" class="investors-img">
+    <div class="investors-content">
+    <h4>${speaker.name}</h4>
+    <h5>${speaker.profession}</h5>
+    <hr class="investors-line" />
+    <p class="investors-description">${speaker.description}</p>
+    </div>
+    </article>`;
+  });
+}
+
+data();
+
+const programCategoriesData = [
+  {
+    icon: './images/phone-view-icons/program_icon_01.png',
+    title: 'Discours',
+    description: 'Listen to the opening talks of the manager about the restaurant.',
+  },
+  {
+    icon: './images/phone-view-icons/program_icon_02.png',
+    title: 'Presentation',
+    description: 'Video presentented on a large screen of diffrerent types of food present in our various menu.',
+  },
+  {
+    icon: './images/phone-view-icons/program_icon_03.png',
+    title: 'Forum',
+    description: 'Listen to diffrerent ways of improving our service to have a better experience.',
+  },
+  {
+    icon: './images/phone-view-icons/program_icon_04.png',
+    title: 'Exchange',
+    description: 'Opportunity open to recrute good personnel.',
+  },
+  {
+    icon: './images/phone-view-icons/program_icon_05.png',
+    title: 'Degustation',
+    description: 'party time and enjoyement will be available',
+  },
+];
+
+const programCategoriesContainer = document.getElementById('program-details-container');
+
+function createprogramCategories() {
+  programCategoriesData.forEach((category) => {
+    programCategoriesContainer.innerHTML += `
+    <div class="programe-detail">
+    <img src="${category.icon}" alt="">
+    <span class="title-programe">${category.title}</span>
+    <p class="programe-para">${category.description}</p>
```

Comment:
- [ Optional ] I think it would be better if you `center` this paragraph the way it's centered on the Behance design. Remember following design guidelines marks you as a developer who can bring out designs the same way they are illustrated in design documents.

- [ Optional ] I think it would be better if you add a `hover` effect that changes the `border color` of the program cards when a user hovers over them just as illustrated in the Behance design.

## Your design
![Screenshot from 2023-07-06 23-12-11](https://github.com/axcell-0/first-capstone/assets/16392046/77eac7b0-bf6a-4001-b13d-4515e4d86598)

## Behance design
![Screenshot from 2023-07-06 23-18-33](https://github.com/axcell-0/first-capstone/assets/16392046/672b541b-e47c-4bf0-8a1e-f1e71fab344c)